ecocode-rules-specifications/src/main/rules/EC31/javascript/EC31.asciidoc

```diff
@@ -6,18 +6,24 @@ Using appropriate image formats and optimizing image sizes is essential for impr
 Larger image file sizes consume more bandwidth, increasing the data transfer required to load a web page.
 Some image formats are generally considered better for eco-friendly web design and should be used in most cases.
 
+== How to fix it
+
+=== Noncompliant code example
+
+[source,html,data-diff-id="1",data-diff-type="noncompliant"]
+----
+<img src="./assets/images/cat.jpg" alt="Unoptimized image of a cat"/> // Noncompliant
+----
+
+=== Compliant solution
+
 We recommend using the following formats:
```

Comment:
Why this section is now in "Compliant solution"?

Reply:
The section change was reverted.